src/libs/actions/Session/index.ts

```diff
@@ -332,28 +332,38 @@ function signInWithShortLivedAuthToken(email: string, authToken: string) {
                 isLoading: true,
             },
         },
-    ];
-
-    const successData: OnyxUpdate[] = [
+        // We are making a temporary modification to 'signedInWithShortLivedAuthToken' to ensure that 'App.openApp' will be called at least once
         {
             onyxMethod: Onyx.METHOD.MERGE,
-            key: ONYXKEYS.ACCOUNT,
+            key: ONYXKEYS.SESSION,
             value: {
-                isLoading: false,
+                signedInWithShortLivedAuthToken: true,
             },
         },
     ];
 
-    const failureData: OnyxUpdate[] = [
+    // Subsequently, we revert it back to the default value of 'signedInWithShortLivedAuthToken' (undefined) in 'successData' or 'failureData' to ensure the user is logged out on refresh
```

Comment:
Why are we setting `signedInWithShortLivedAuthToken` to `undefined` specifically, rather than `null` so it gets removed?

Reply:
I actually don't have a specific strong reason for this, we can use null if you feel it's better to remove it rather than keeping it as undefined.

Reply:
I think we should just switch this to `null`. I missed this bit.

Reply:
Done